src/applications/personalization/dashboard/components/DashboardAlert.jsx

```diff
@@ -26,17 +27,20 @@ const DashboardAlert = ({
   );
 
   const alertContent = content || children;
+  const headerId = `dashboard-alert-header-${id}`;
 
   return (
-    <div className={classes}>
-      <header>
+    <div aria-labelledby={headerId} role="region" className={classes}>
+      <div>
```

Comment:
is switching this from a `header` to `div` for a11y reasons or other reasons?

Reply:
It was in Trevor's recommendation in the linked issue, I think it's just that it's a confusing landmark when navigating through the page. Not fully sure, though.

Reply:
@erikphansen && @jbalboni Changing to a div was so assistive device users wouldn't have two headers in their landmarks menu. It was a bit confusing when I was navigating by landmarks, and heard a second header announced inside the main content area.